### ISSUE TITLE
How to swap just two items, using drag and drop function

### ISSUE THREAD

**Issue by Card2Me** [h6ah4i/android-advancedrecyclerview#253]
Hi.

I want to swap just two items. basically the grid example rearrange items between two items.

any idea?

**Comment by h6ah4i** [comment]
Hi. Sorry, this library does not support such behavior and I do not have a plan to support that.

**Comment by h6ah4i** [comment]
UPDATE: Okay, I will try to implement it. Some others are requesting this feature. (#269)

**Comment by h6ah4i** [comment]
@Card2Me, @TapanHp I have just made a solution of this issue. I added the `setItemMoveMode()` to `RecyclerViewDragDropManager`, it allows to configure the behavior as "swap" mode. This new feature will be included into the next release, but you can try it now using JitPack.

---

Tip: try develop branch library via JitPack.

```gradle
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:c9763e000fa8e80e6410f4a9d500e21b0e58a7e5'
}
```

---

![demo](https://cloud.githubusercontent.com/assets/2552365/16540741/ed7d753e-40a9-11e6-9d5a-cbb1412844af.gif)

**Comment by TapanHp** [comment]
What you did man ? Its awesome that's what i needed .. You are awesome seriously. Thanks for making this functionality available. Your library was and will be so helpful , Thanks.

**Comment by h6ah4i** [comment]
@TapanHp Hi. I have just merged the develop branch and released it as v0.9.2. It includes the `setItemMoveMode()` method. Thanks 👍
